src/components/BuildBaronAndAnnotations/AnnotationNote.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import { useMutation } from "@apollo/client";
+import styled from "@emotion/styled";
+import Button, { Variant } from "@leafygreen-ui/button";
+import { Disclaimer } from "@leafygreen-ui/typography";
+import { Input, Tooltip } from "antd";
+import {
+  EditAnnotationNoteMutation,
+  EditAnnotationNoteMutationVariables,
+  Note,
+} from "gql/generated/types";
+import { useBannerDispatchContext } from "../../context/banners";
+import { EDIT_ANNOTATION_NOTE } from "../../gql/mutations/edit-annotation-note";
+import { getDateCopy } from "../../utils/string";
+import { ConditionalWrapper } from "../ConditionalWrapper";
+import { TicketsTitle, TitleAndButtons, MetaDataWrapper } from "./BBComponents";
+
+interface Props {
+  note: Note;
+  taskId: string;
+  execution: number;
+  userCanModify: boolean;
+}
+
+export const AnnotationNote: React.FC<Props> = ({
+  note,
+  taskId,
+  execution,
+  userCanModify,
+}) => {
+  const originalMessage = note?.message;
+  const dispatchBanner = useBannerDispatchContext();
+  const [newMessage, setMessage] = useState(originalMessage);
+  const [
+    updateAnnotationNote,
+    { loading: loadingAnnotationNote },
+  ] = useMutation<
+    EditAnnotationNoteMutation,
+    EditAnnotationNoteMutationVariables
+  >(EDIT_ANNOTATION_NOTE, {
+    onCompleted: () => {
+      dispatchBanner.successBanner(`Annotation note updated.`);
+    },
+    onError(error) {
+      dispatchBanner.errorBanner(
+        `There was an error updating this note: ${error.message}`
+      );
+    },
+    refetchQueries: ["GetTask"],
+  });
+  const saveAnnotationNote = () => {
+    updateAnnotationNote({
+      variables: {
+        taskId,
+        execution,
+        originalMessage,
+        newMessage,
+      },
+    });
+  };
+
+  return (
+    <TitleAndButtons>
+      <TicketsTitle>Note</TicketsTitle>
+      {note && (
+        <>
```

Comment:
This Fragment isn't necessary since `<MetaDataWrapper>` is already wrapping all the content here.